plugins/ashby/src/App.css

```diff
@@ -0,0 +1,228 @@
+/* Your Plugin CSS */
+
+main {
+    display: flex;
+    flex-direction: column;
+    align-items: start;
+    padding: 0 15px 0;
+    height: 100%;
+    gap: 15px;
+
+    user-select: none;
+    -webkit-user-select: none;
+}
```

Comment:
Overriding the tint color will make checkboxes, input focus borders, and primary buttons use the Ashby accent color instead of the default blue color.

I think only the first color will actually be visible in this plugin because it has no primary buttons (the darkened variants are hover and pressed state background colors).

<img width="333" height="123" alt="image" src="https://github.com/user-attachments/assets/0b2b1d27-c23e-438b-8aa1-e029d1f76645" />

```suggestion
	--framer-color-tint: #5f55cc;
	--framer-color-tint-dark: #574dbd;
	--framer-color-tint-extra-dark: #5047b3;
}
```